sdk/src/main/java/com/talkable/sdk/WebAppInterface.java

```diff
@@ -71,6 +71,10 @@ public void publish(String eventName, String data) {
                     talkableOfferFragment.shareOfferViaTwitter(
                             JsonUtils.getJsonString(json, "message"));
                     break;
+                case "share_offer_via_whatsapp":
+                    talkableOfferFragment.shareOfferViaWhatsapp(
+                            JsonUtils.getJsonString(json, "message"));
```

Comment:
Why claim_url is not passed for Twitter and WhatsApp, but passed for other channels?

Reply:
It is a question not to the SDK but to `talkable.js`. It passes only `message` for that sharing methods.

Reply:
And actually message contains claim URL. Here is a sample message from default theme: `Get $5 off your first order at dgt-store https://test-prod2.automatic-tests-talkable.com/x/6lLb5Z`